lib/tasks/deploy.rake

```diff
@@ -0,0 +1,50 @@
+namespace :custom_assets do
+  # Downloads a tar from a URL, extracts its contents into ./tmp,
+  # and returns a colon separated list of paths belonging to the first level directories
+  # inside the original tar file.
+  desc "Download external assets"
+  task :download do
+    Bundler.require :default, :development
+
+    url = ENV['_URL']
+    branch = ENV['_BRANCH']
+    credentials = ENV['_CREDENTIALS']
```

Comment:
is there a reason these are all underscore prefixed? should they maybe be `ASSETS_URL` or `ASSETS_BRANCH` etc?